Sources/Charcoal/Filters/Map/MapFilterViewController.swift

```diff
@@ -141,14 +152,6 @@ public class MapFilterViewController: FilterViewController {
 
         navigationItem.rightBarButtonItem = toggleButton
 
-        if let bboxFilter = bboxFilter,
-            let polygonFilter = polygonFilter,
-            selectionStore.isSelected(polygonFilter) || selectionStore.isSelected(bboxFilter) {
-            selectedViewController = mapPolygonFilterViewController
-        } else {
-            selectedViewController = mapRadiusFilterViewController
-        }
```

Comment:
We can't remove this code, it makes sure if you already have a filter selected, it will display that filter.
Now there's a problem that can be reproduced like this:
1. Open realestate search (it shows polygon map by default), but switch over to radius search
2. Make a search with radius filter
3. Close bottom sheet
4. Open bottom sheet, click map filter
5. Problem: Now the map filter displays polygon search, but radius search is the active filter

I think we can fix this by using the default mode if you have neither a polygon, bbox or radius filter selected (instead of using radius in that case).
Then we might be able to remove the selection of view controller from init as well, will make it simpler if it's only here.

Reply:
i think we'll need it in the init also, as selectedViewController needs to be instantiated before function calls (self). or is there a good way to set it outside the init?